docs/private-app-create.md

```diff
@@ -0,0 +1,92 @@
+## Glossary
+
+|Term|Description|
+|------|-------------|
+|Private App|An application/service/ETL tool that is developed by a user who wants to be able to programmatically interact with the Gateway. |
+|AppID and ClientID |Tokens that are generated when creating a new Private App. Each Private App will have a unique AppID and ClientID that needs to be sent with each API request to authenticate and approve the action. |
+|Team Admin/Developer |A role that can be assigned to a user when they are part of a ‘team’ on the Gateway. Please see the documentation [here](https://www.healthdatagateway.org/about/how-to-manage-teams-on-the-gateway) on roles and permissions on the Gateway. |
+|Authenticated API Endpoint |While a lot of the Gateway APIs are accessible without authentication, any endpoint changing data requires authentication. The ClientID acts as an authentication token. |
+
+## Introduction
+
+The Private Apps feature enables developers to create a Private App and corresponding ClientID which is used as a token to authenticate requests made to the Gateway API. This guide offers step-by-step instructions for setting up and managing Private Apps on the Gateway. 
```

Comment:
Capitalise 'Developer's to indicate it is the role (with associated permissions) on the Gateway team

Reply:
Adjusted, in next merge